src/main/java/com/cvsgo/repository/ProductCustomRepositoryImpl.java

```diff
@@ -222,4 +230,16 @@ private BooleanExpression priceGreaterOrEqual(Integer lowestPrice) {
         return lowestPrice != null ? product.price.goe(lowestPrice) : null;
     }
 
+    private BooleanBuilder searchKeyword(String keyword,
+        StringExpression productAndManufacturerName) {
+        BooleanBuilder builder = new BooleanBuilder();
+        if (keyword != null) {
+            String[] keywords = keyword.replace(" ", "").split("");
```

Comment:
공백을 다 없앤 후에 split()을 하면 검색어가 "펩시 콜라"인 경우, 한 글자씩 ["펩", "시", "콜", "라"] 이렇게 쪼개지게 되는데요,
["펩시", "콜라"] 이렇게 split 되는걸 의도하신거라면 이렇게 해야 할 것 같습니다!
```suggestion
            String[] keywords = keyword.split(" ");
```

Reply:
저는 `불닭컵` 검색 결과와 `컵불닭`검색 결과가 동일하게 나와야한다고 생각해서 한 글자씩 쪼개지는 로직으로 구현했습니다!